sklearn/semi_supervised/label_propagation.py

```diff
@@ -17,8 +17,8 @@
   The algorithm tries to learn distributions of labels over the dataset. In the
   "Hard Clamp" mode, the true ground labels are never allowed to change. They
```

Comment:
I've made the description a bit more generic and about _variants_. `LabelPropagation` and `LabelSpreading` are examples from these different variants.